spec/deployment_base_spec.rb

```diff
@@ -2,6 +2,26 @@
 
 RSpec.describe OpenStax::Aws::DeploymentBase do
 
+  context "env_name" do
+    it 'doesn\'t match regex' do 
+      deployment_class = Class.new(described_class) do
+        template_directory __dir__, 'support/templates/factory_test'
+
+        secrets :common do
+          specification do
+            content do
+              { foo: "blah" }
+            end
+          end
+        end
+      end
```

Comment:
don't think you need this `secrets` declaration.  if you do, should use another simpler test template.  should remove so people don't think it is important.